package.json

```diff
@@ -49,6 +50,7 @@
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint . --cache --ext js --ext md --ext ts --report-unused-disable-directives",
     "lint:md": "markdownlint --ignore **/CHANGELOG.md packages/**/*.md README.md",
+    "new:rule": "cd packages && node sonarwhal/dist/src/bin/sonarwhal.js --new-rule",
```

Comment:
Again, I think we should run the build for `sonarwhal` before executing `node sonarwhal/dist/src/bin/sonarwhal.js --new-rule`.